docs/_articles/ja/user-guide/perf-enhancements.md

```diff
@@ -14,4 +14,9 @@ To enable this beta feature:
 1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
 1. Under Salesforce Feature Previews, select Experimental: Deploy Retrieve.
 
-In this beta release, performance enhancements will only occur when running the **SFDX: Deploy this Source to Org** command from an Apex Class.
+In this beta release, performance enhancements will only occur when running the **SFDX: Deploy this Source to Org** command with the following metadata types:
```

Comment:
Consider rephrasing:
In this beta release, enhanced code performance is effective when you run the **SFDX: Deploy this Source to Org** command with these metadata types: